test cases/fortran/9 cpp/main.cpp

```diff
@@ -2,7 +2,7 @@
 
 extern "C" double fortran();
 
-int main(int, char**) {
-    std::cout << "FORTRAN gave us this number: " << fortran() << '\n';
-    return 0;
+int main(void) {
+    std::cout << "FORTRAN gave us this number: " << fortran() << std::endl;
```

Comment:
These changes are not according to contemporary recommended C++ code. Leaving out argument names is the standard for not used arguments, and `std::endl` is recommended against unless it is explicitly needed (which is not the case here).